jupiterone/internal/client/rule.graphql

```diff
@@ -88,7 +146,6 @@ mutation CreateReferencedQuestionRuleInstance(
 # `templates: null` will not when include in the request.
 # @genqlient(for: "RuleOperationInput.when", omitempty: true)
 # @genqlient(for: "UpdateInlineQuestionRuleInstanceInput.latestAlertId", omitempty: true)
-# @genqlient(for: "UpdateInlineQuestionRuleInstanceInput.outputs", omitempty: true)
```

Comment:
outputs cannot be omitted, otherwise outputs will not serialize correctly to: outputs = null, when a user desires to enable and then later disable outputs for the rule resource. I tested this directly against our graphql api to observe the correct and intended format that the api expects.